app.js

```diff
@@ -146,15 +153,25 @@ function pingCowin({ key, hook, age, districtId, appointmentsListLimit, date, pi
             if (hook && key) {
                 axios.post(`https://maker.ifttt.com/trigger/${hook}/with/key/${key}`, { value1: dataOfSlot }).then(() => {
                     console.log(dataOfSlot);
-                    console.log('Sent Notification to Phone \nStopping Pinger...')
-                    sound.play(notificationSound);
-                    clearInterval(timer);
+                    if (keepAlive) {
+                        console.log('Sent Notification to Phone')
+                        sound.play(keepAliveNotificationSound);
+                    } else {
+                        console.log('Sent Notification to Phone \nStopping Pinger...')
+                        sound.play(notificationSound);
+                        clearInterval(timer);
+                    }
                 });
             } else {
                 console.log(dataOfSlot);
-                console.log('Slots found\nStopping Pinger...')
-                sound.play(notificationSound, 1);
-                clearInterval(timer);
+                if (keepAlive) {
+                    console.log('Slots found')
+                    sound.play(keepAliveNotificationSound, .8);
```

Comment:
This notification sound is a little too short and would be easy to miss. Let's keep the existing one itself or  a longer notification sound if you can find one.

Reply:
I added a new tone to differentiate keep alive flow and normal flow. I'll revert it back to the original, that makes more sense